web-common/src/main/java/edu/unc/lib/boxc/web/common/services/AccessCopiesService.java

```diff
@@ -51,7 +51,8 @@ public List<ContentObjectRecord> listViewableFiles(PID pid, AccessGroupSet princ
         ContentObjectRecord briefObj = solrSearchService.getObjectById(new SimpleIdRequest(pid, principals));
         String resourceType = briefObj.getResourceType();
         if (ResourceType.File.nameEquals(resourceType)) {
-            if (briefObj.getDatastreamObject(DatastreamType.JP2_ACCESS_COPY.getId()) != null) {
+            if (briefObj.getDatastreamObject(DatastreamType.JP2_ACCESS_COPY.getId()) != null
+                    || briefObj.getDatastreamObject(DatastreamType.AUDIO_ACCESS_COPY.getId()) != null) {
```

Comment:
I'm not totally sure what the impact of this is going to be until we test it, just making a note to remember to check back in on it